core/src/components/refresher/test/basic/e2e.ts

```diff
@@ -1,10 +1,28 @@
+import type { E2EPage } from '@stencil/core/testing';
 import { newE2EPage } from '@stencil/core/testing';
 
-test('refresher: basic', async () => {
-  const page = await newE2EPage({
-    url: '/src/components/refresher/test/basic?ionic:_testing=true'
+import { pullToRefresh } from '../test.utils';
+
+describe('refresher: basic', () => {
+
+  let page: E2EPage;
+
+  beforeEach(async () => {
+    page = await newE2EPage({
+      url: '/src/components/refresher/test/basic?ionic:_testing=true'
+    });
+  });
+
+  it('should match existing visual screenshots', async () => {
+    const compare = await page.compareScreenshot();
+    expect(compare).toMatchScreenshot();
+  });
+
+  it('should load more items when performing a pull-to-refresh', async () => {
+    await pullToRefresh(page);
+
+    const items = await page.findAll('ion-item');
+    expect(items.length).toBe(60);
```

Comment:
Can we check the number of items before the PTR to verify that it does add new items?